lms/static/sass/course/instructor/_instructor_2.scss

```diff
@@ -956,7 +973,148 @@ section.instructor-dashboard-content-2 {
       }
     }
   }
+
+  /*
```

Comment:
@clrux, where do we stand with this override being needed/still relevant. I'd like to be able to view the actual Sass you are planning on introducing into the final production code. Does that mean this needs to be refactored/removed?